README.md

```diff
@@ -423,6 +422,65 @@ resp := &response{}
 err := c.RequestWithContext(ctx, "foo", req, resp)
 ```
 
+## JetStream Basic Usage
+
+```go
+import nats "github.com/nats-io/nats.go"
+
+// Connect to NATS
+nc, _ := nats.Connect(nats.DefaultURL)
+
+// Connect to JetStream
+js, _ := nc.JetStream()
+
```

Comment:
Show both management and normal as separate sections?

Reply:
Split into 2 sections: JetStream Basic Usage and JetStream Basic Management.